src/System.Reflection.Metadata/specs/PE-COFF.md

```diff
@@ -36,3 +36,20 @@ If the containing PE/COFF file is deterministic the Guid field above and DateTim
 The entry doesn't have any data associated with it. All fields of the entry, but Type shall be zero.
 
 Presence of this entry indicates that the containing PE/COFF file is deterministic. 
+
+### Embedded Portable PDB Debug Directory Entry (type 17)
+
+Declares that debugging information is embedded in the PE file at location specified by PointerToRawData. 
+
+*Version Major=any, Minor=0x0100* of the data format:
+
+| Offset | Size           | Field            | Description                                           |
+|:-------|:---------------|:-----------------|-------------------------------------------------------|
+| 0      | 4              | UncompressedSize | The size of decompressed Portable PDB image           |
+| 4      | SizeOfData - 4 | PortablePdbImage | Portable PDB image compressed using Deflate algorithm | 
+
+If this entry is present and the reader recognizes this entry the debugging information for the PE file shall be read from the embedded data. If a CodeView entry is also present it shall be ignored. 
+
+> Note: Including both entries enables a tool that does not recognize Embedded Portable PDB entry to locate debug infomration as long as it is also available in a file specified in CodeView entry. Such file can be created by extracting the embedded Portable PDB image to a separate file.
+
+The Major version indicates the version of the Portable PDB format itself. The Minor version indicates the version of the Embedded Portable PDB data format.
```

Comment:
It has the version. See above: https://github.com/dotnet/corefx/pull/10199/files/e1dd448af8a04c960377d3b8071f85b4ad43186a#diff-f595b39db5266b534bf46339a9f285efR32